app/Http/Controllers/EmployeeController.php

```diff
@@ -39,10 +43,50 @@ public function edit(Employee $employee)
     {
         return view('employee.edit', compact('employee'));
     }
-    public function update(EmployeeRequest $request,Employee $employee)
+    public function update(EmployeeRequest $request, Employee $employee)
     {
         $employee->update($request->validated());
         return to_route('index')->with('message', 'Employee Succesfully Updated');
+    }
+    public function show_wallet($id)
+    {
+        $wallet = Wallet::where('employee_id', $id)->first();
+        $wallet_date = Carbon::parse($wallet->last_credited)->format('d-m-Y');
+        $wallet_time = Carbon::parse($wallet->last_credited)->toTimeString();
```

Comment:
You either change the property in the db to use date time or you cast these columns to use the desired format you want on their model. Do this so you don't have to continuously format these values before rendering. This helps to speed up execution.